lighthouse-core/test/report/proto-test.js

```diff
@@ -6,12 +6,21 @@
 'use strict';
 
 const sampleJson = require('../results/sample_v2.json');
-const roundTripJson = require('../../../proto/sample_v2_round_trip.json');
 const preprocessor = require('../../lib/proto-preprocessor.js');
 
 /* eslint-env jest */
 
-describe('round trip JSON comparison subsets', () => {
+let roundTripJson;
```

Comment:
> I am a little nervous that this is going to get tripped up and stop running at some point and we won't notice since the tests will just get skipped

ah, ok, what do you think about something like
```js
let roundTripJson;
let describeIfProtoExistsOrWhatever;
try {
  roundTripJson = require('../../../proto/sample_v2_round_trip.json');
  describeIfProtoExistsOrWhatever = describe;
} catch (err) {
  // `yarn test-proto` runs in Actions CI, so throw if this test somehow won't be running there.
  if (!process.env.GITHUB_ACTIONS) {
    throw new Error('sample_v2_round_trip must be generated for CI proto test');
  }
  // Otherwise no proto roundtrip for tests, so skip them.
  describeIfProtoExistsOrWhatever = describe.skip;
}
```
It's a little more verbose than the ternary, but by setting `describe` in the try/catch check it ensures they don't get separated.

Reply:
cool, done. I also moved this to test-utils.